hood/cli/account.py

```
@@ -6,5 +6,5 @@ def accounts():
     credential = _util.get_credential()
     client = CryptoTradingClient(credential)
 
-    result, err = client.accounts(create_namespace=True)
+    result, err = client.accounts()
```

Ruff (0.14.10)

Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)